calicoctl/commands/delete.go

```diff
@@ -46,7 +46,10 @@ Options:
   -s --skip-not-exists      Skip over and treat as successful, resources that
                             don't exist.
   -f --filename=<FILENAME>  Filename to use to delete the resource.  If set to
-                            "-" loads from stdin.
+                            "-" loads from stdin. If filename is a directory, this command is
+                            invoked for each .json .yaml and .yml file within that directory,
+                            terminating after the first failure
```

Comment:
I wasn't going to comment on all of the missing `.`, but here I am.

Reply:
hahahaha